app/controllers/todos_controller.rb

```diff
@@ -16,7 +19,9 @@ def new
 
   def create
     @todo = Todo.new(todo_params)
+    @todo.user = current_user

```

Comment:
Definitely not on the critical path, but we thought using `current_user.todos.build` (given by the has_many association) was pretty nifty. It populates the todo's user_id field automatically, which saves you from having to set it manually.

Reply:
Great idea! There's a lot of wonderful Rails tricks to remember to use.